src/compiler/transformers/ts.ts

```diff
@@ -18,6 +18,22 @@ namespace ts {
         NonQualifiedEnumMembers = 1 << 3
     }
 
+    const enum ClassFacts {
+        HasStaticInitializedProperties = 1 << 0,
+        HasConstructorDecorators = 1 << 1,
+        HasMemberDecorators = 1 << 2,
+        IsNamespaceExport = 1 << 3,

```

Comment:
`IsExportOfNamespace` would make more sense, since this isn't the counterpart to a namespace import (`import * as foo from "foo";`).